src/app/home/home.component.html

```diff
@@ -7,11 +7,7 @@
 <md-tab-group>
     <md-tab label="Games">
 
-        <md-card
-            class="game-card"
-            *ngFor="let game of user.games"
-            [routerLink]="['/games', game.id]"
-        >
+        <md-card class="game-card" *ngFor="let game of user.games" [routerLink]="['/games', game.id]">
```

Comment:
Please change formatting only if you change the code. (otherwise, you'll mess with git blame, git history...)